Zeit.co now previews are failing

### Steps to reproduce the problem
0. You need to be part of Zeit Qiskit organization. If you think you should be there, ask @delapuente 
1. Perform a PR to master.

### What is the current behavior?
Preview deployment is failing. This is a dump of the logs:

```
Downloading 1217 deployment files...
Installing build runtime...
Build runtime installed: 531.276ms
Looking up build cache...
Build cache found [65.99 MB, 38543 files]
Build cache unpacked: 2594.615ms
Installing dependencies...
npm WARN webpack-dev-server@2.11.5 requires a peer of webpack@^2.2.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN webpack-dev-middleware@1.12.2 requires a peer of webpack@^1.0.0 || ^2.0.0 || ^3.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/webpack-dev-server/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/watchpack/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.9 (node_modules/jest-haste-map/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

added 430 packages from 854 contributors and updated 5 packages in 12.037s

 FATAL  Cannot find module './hooks/generate-textbook-toc'
Require stack:
- /zeit/189d3eb5/nuxt.config.js
- /zeit/189d3eb5/node_modules/@nuxt/cli/dist/cli-command.js
- /zeit/189d3eb5/node_modules/@nuxt/cli/dist/cli.js
- /zeit/189d3eb5/node_modules/nuxt/bin/nuxt.js

  Error: Cannot find module './hooks/generate-textbook-toc'
  Require stack:
  - nuxt.config.js
  - node_modules/@nuxt/cli/dist/cli-command.js
  - node_modules/@nuxt/cli/dist/cli.js
  - node_modules/nuxt/bin/nuxt.js
  at new MODULE_NOT_FOUND (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at wu (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Eu (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at ku (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Proxy.<anonymous> (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Object.<anonymous> (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Tu (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Mu (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Object.import (/zeit/5616d15c/node_modules/esm/esm.js:1)
  at Object.<anonymous> (nuxt.config.js:1)

tput: No value for $TERM and no -T specified

   ╭─────────────────────────────────────────────────────────────────╮
   │                                                                 │
   │   ✖ Nuxt Fatal Error                                            │
   │                                                                 │
   │   Error: Cannot find module './hooks/generate-textbook-toc'     │
   │   Require stack:                                                │
   │   - /zeit/189d3eb5/nuxt.config.js                               │
   │   - /zeit/189d3eb5/node_modules/@nuxt/cli/dist/cli-command.js   │
   │   - /zeit/189d3eb5/node_modules/@nuxt/cli/dist/cli.js           │
   │   - /zeit/189d3eb5/node_modules/nuxt/bin/nuxt.js                │
   │                                                                 │
   ╰─────────────────────────────────────────────────────────────────╯

Error: Exited with 1
    at ChildProcess.child.on (/zeit/60065632285cb81c/.build-utils/.builder/node_modules/@now/static-build/dist/index.js:31782:24)
    at emitTwo (events.js:126:13)
    at ChildProcess.emit (events.js:214:7)
    at maybeClose (internal/child_process.js:925:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
worker exited with code 20 and signal null
done
```

### What is the expected behavior?
Nothing should fail and the Zeit.co preview should be available.

--- a/.eslintignore
+++ b/.eslintignore
@@ -89,6 +89,3 @@ dist
 
 # Service worker
 sw.*
-
-# Build dir
-public

--- a/.gitignore
+++ b/.gitignore
@@ -86,6 +86,3 @@ dist
 
 # Service worker
 sw.*
-
-# Build dir
-public

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -35,7 +35,7 @@ Run unit tests made with [Jest](https://jestjs.io/):
   npm run test
 ```
 
-Build static version ready for production, output will generated inside a new folder called `public`:
+Build static version ready for production, output will generated inside a new folder called `dist`:
 ```shell
   npm run build
 ```

--- a/README.md
+++ b/README.md
@@ -163,7 +163,7 @@ Run unit tests made with [Jest](https://jestjs.io/):
   npm run test
 ```
 
-Build static version ready for production, output will generated inside a new folder called `public`:
+Build static version ready for production, output will generated inside a new folder called `dist`:
 ```shell
   npm run build
 ```

--- a/deploy/deploy_cos.sh
+++ b/deploy/deploy_cos.sh
@@ -10,6 +10,6 @@ CONFIG_PATH=$(rclone config file | tail -1)
 openssl aes-256-cbc -K $encrypted_rclone_key -iv $encrypted_rclone_iv -in rclone.conf.enc -out $CONFIG_PATH -d
 
 git clone --depth=1 https://github.com/Qiskit/qiskit-terra.git
-cp -r qiskit-terra/qiskit/schemas ../public/.
+cp -r qiskit-terra/qiskit/schemas ../dist/.
 
-rclone sync --exclude 'documentation/**' ../public IBMCOS:qiskit-org-website
+rclone sync --exclude 'documentation/**' ../dist IBMCOS:qiskit-org-website

--- a/nuxt.config.ts
+++ b/nuxt.config.ts
@@ -171,7 +171,6 @@ const config: Configuration = {
   },
 
   generate: {
-    dir: 'public',
     routes: (function () {
       const events = fs.readdirSync(path.resolve(__dirname, 'content', 'events'))
         .filter(filename => path.extname(filename) === '.md')

